presto-cli/src/main/java/io/prestosql/cli/ClientOptions.java

```diff
@@ -214,6 +218,15 @@ public static URI parseServer(String server)
         return builder.build();
     }
 
+    public static Map<String, String> toCredentials(List<ClientExtraCredentials> credentials)
+    {
+        ImmutableMap.Builder<String, String> builder = ImmutableMap.builder();
+        for (ClientExtraCredentials credential : credentials) {
+            builder.put(credential.getName(), credential.getValue());
+        }
+        return builder.build();
```

Comment:
```
return credentials.stream()
   .colect(toImmutableMap(...));
```

Reply:
I am OK with it using `stream()` API, but the code style of the whole class looks weird as similar functions like `toResourceEstimates()` and `toProperties` use the for-each loop.

Reply:
I agree, it’s better to keep the style consistent. Thanks for noticing.

Reply:
I would also change other methods here (as separate commit or PR). Up to you.

Reply:
Refactor can be done in subsequent PR.